maint/gen_ch4_api.py

```diff
@@ -185,7 +185,7 @@ def dump_netmod_impl_h(h_file):
 
             print("{", file=Out)
             use_ret = ''
-            if re.search(r'int|size_t', a['ret']):
+            if re.search(r'int|size_t|bool|void\s\*', a['ret']):
```

Comment:
It is still not scalable if we add more return types. How about change into:
```
if re.match(r'void\s*$', a['ret']):
```

Reply:
Fixed. Can you please check again?